client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -549,11 +549,29 @@ Undocumented.prototype.isDomainAvailable = function( domain, fn ) {
 };
 
 /**
+ * Get the inbound transfer status for this domain
+ *
+ * @param {string} domain - The domain name to check.
+ * @param {Function} fn The callback function
+ * @returns {Promise} A promise that resolves when the request completes
+ * @api public
+ */
+Undocumented.prototype.getInboundTransferStatus = function( domain, fn ) {
+	return this.wpcom.req.get(
+		{
+			path: `/domains/${ encodeURIComponent( domain ) }/inbound-transfer-status`,
```

Comment:
Be sure to change this on the backend too :) Though, looking now on the information that this endpoint returns, maybe `info` fits better - up to you. Just make it consistent between fronted and backend ;)